pymysql/connections.py

```diff
@@ -890,6 +919,12 @@ def _execute_command(self, command, sql):
         if isinstance(sql, text_type):
             sql = sql.encode(self.encoding)
 
+        local_file_name = re.search(b'load data local infile \'(?P<file_name>[^\']+)\'', sql, flags=re.IGNORECASE)
+        if local_file_name:
```

Comment:
I feel this check is too much.
This expression doesn't match when there are TABs or LFs.
Since users should enable `local_infile` option only when they believe server, I'd like to completely remove this check.